### PR TITLE
[Downgrade PHP 7.1] iterable pseudo-type

### PR DESCRIPTION
**Pull request by norberttech**
solves #4195

### CODE REVIEW

File: docs/rector_rules_overview.md [hunk 1]
```diff
@@ -4531,6 +4531,48 @@ return static function (ContainerConfigurator $containerConfigurator): void {
...
+↓
+
+```diff
+ <?php
+
+ class SomeClass
+ {
+-    public function run(iterable $iterator)
++    /**
++     * @param mixed[]|\Traversable $iterator
```
Comment (norberttech) [inline docs/rector_rules_overview.md:4565]: the original issue says it should be `@param array|\Traversable $iterator` instead but `mixed[]` seems a bit more specific, please let me know which one is preferred 😊

Comment (norberttech) [reply docs/rector_rules_overview.md:4565]: resolved according to the https://github.com/rectorphp/rector/issues/4195#issuecomment-730049526

**Comment by norberttech** [comment]
hey, is there anything I should fix here? (beside resolving the conflicts obviously)

**Comment by TomasVotruba** [comment]
Hi,

the CI should be green. It's failing on 5 place now.
I can review deply then and get this merged if allright.

**Comment by norberttech** [comment]
right, sorry, it looks like something came after rebase, I'm closing this, the CI was fine a week ago, I'm currently a bit busy with my own stuff, I'm sure someone else will pick this one up later

**Comment by TomasVotruba** [comment]
No problem, I'll give it 10 minutes and see it that's enough

**Comment by TomasVotruba** [comment]
Merged in https://github.com/rectorphp/rector/pull/4649, thank you :+1:

**Pull request closed without merging**